### DISCUSSION TITLE
RAP Smoke and Particulate matter variables

### DISCUSSION THREAD

**Discussion by moab360** [NOAA-GSL/GSL-Regional-Model-Forum#44] [General]
I noticed that the particulate matter variables in the RAP model seem to be handled independently of the columnar smoke particulate mass.  Just a basic glance at the grib data doesn't show any clear correlation between where there is smoke and the particulate matter.  Could you please confirm whether these variables (particulate matter fine and particulate matter course) are coupled with or independent from the smoke variable in the model?

**Comment by EricJames-NOAA** [comment]
@moab360 the smoke particulate matter (MASSDEN) fields in RAP are smoke, from biomass burning.  Smoke is explicitly simulated in the RAP (and HRRR), with a plume rise parameterization, smoke transport, and wet and dry removal.  However, the fine and coarse particulate matter (PMTF and PMTC) are climatological aerosols from the Thomson-Eidhammer aerosol-aware microphysics scheme.  These are much less realistic but are needed for the microphysics scheme.  Please let us know if you have other questions about this.

**Reply by EricJames-NOAA** [reply]
I should also mention that the COLMD fields are based on the MASSDEN fields.  These are all calculated from the 3-D distribution of the smoke tracer.

**Comment by moab360** [comment]
Thanks for the info @EricJames-NOAA, and understood about the COLMD being based on the MASSDEN fields.  I was trying to see if there was a way to build out an AQI forecast based on the smoke data, but if the PMTF and PMTC are a generic scheme and not coupled with the smoke particulate, that may be difficult unless there is another variable(s) I missed classifying the smoke particulate size?

**Reply by EricJames-NOAA** [reply]
@moab360 the smoke (MASSDEN) is intended to reflect smoke PM2.5.  So an AQI forecast could be based on this, but it would be missing other components like dust and anthropogenic pollution, for example.

**Reply by moab360** [reply]
Thanks for the info @EricJames-NOAA, very interesting!

**Reply by afernandezody** [reply]
Hi @**EricJames-NOAA**, 
This is an interesting discussion started by @**moab360**. I was wondering if you could provide a bit more info about a couple of topics:

1. Does RRFS use the same model? I'm seeing 'hourly wildfire potential' and 'Smoke emissions' listed among the fields at https://rapidrefresh.noaa.gov/RRFS/. Is it still the same notation (MASSDEN) for PM2.5? Which units is it using?
2. I don't recall any of this for either NAM or GFS? Am I mistaken or did I miss anything?
 
Thanks.

**Reply by EricJames-NOAA** [reply]
@afernandezody to answer your questions:

(1) RRFS also includes prediction of smoke and dust using a similar approach.  Smoke PM2.5 is in this field:

MASSDEN Mass Density (Concentration) [kg/m^3]:aerosol=Particulate organic matter dry:aerosol_size <2.5e-06:

We also have fine dust and coarse dust in these fields:

MASSDEN Mass Density (Concentration) [kg/m^3]:aerosol=Dust dry:aerosol_size <2.5e-06:
MASSDEN Mass Density (Concentration) [kg/m^3]:aerosol=Dust dry:aerosol_size >=2.5e-06,<1e-05:

The units for all of these GRIB2 fields is kg/m3.

(2) Neither NAM nor GFS has a smoke or dust component, to my knowledge.
